packages/core/cms/faststore/sections.json

```diff
@@ -2167,7 +2167,7 @@
         "editLabel": {
           "title": "Location edit label",
           "type": "string",
-          "default": "Edit"
+          "default": ""
```

Comment:
question: Do we want to change those defaults (for `editLabel` and `buttonIcon.icon`/`buttonIcon.alt`? New stores even without DP would have those changes, and stores with DP that don't use global filter. Was it a design decision?